runtimes/eden/src/lib.rs

```diff
@@ -103,8 +103,8 @@ construct_runtime! {
 		ParachainInfo: parachain_info = 31,
 		CumulusXcm: cumulus_pallet_xcm = 32,
 		DmpQueue: cumulus_pallet_dmp_queue = 33,
-		// XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 34,
-		// PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 35,
+		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 34,
+		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 35,
```

Comment:
If we keep to use Eden runtime for Paradis test net as well, then perhaps we need to call this RelayChainXCM. But we will decide about it later.

Reply:
Looks like Karura on Kusama has used PolkadotXCM which probably means pallet_xcm from Polkadot is used for parity's relaychains other than Polkadot too.